### PR TITLE
Working all artists

### PR DESCRIPTION
**Pull request by euniceyoh**

### CODE REVIEW

File: backend/dbUtils.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+const neo4j = require("neo4j-driver");
+const driver = neo4j.driver("neo4j+s://463ded1b.databases.neo4j.io:7687",neo4j.auth.basic("can", "(ABCxyz98)"));
```
Comment (euniceyoh) [inline backend/dbUtils.js:3]: i think we should create a .env file to store credentials

Comment (euniceyoh) [inline backend/dbUtils.js:3]: i think we should use a .env file to store credentials

Comment (canhuynh1998) [reply backend/dbUtils.js:3]: yes, will do it ASAP. I totally forgot about this, thanks!

Comment (euniceyoh) [reply backend/dbUtils.js:3]: cool. i used a .env when working on my feature too so if we merge all our of stuff eventually, we should have one too. either way works

**Pull request merged**